src/material/schematics/ng-update/migrations/theming-api-v12/theming-api-migration.ts

```diff
@@ -24,7 +28,19 @@ export class ThemingApiMigration extends DevkitMigration<null> {
         this.fileSystem.edit(stylesheet.filePath)
           .remove(0, stylesheet.content.length)
           .insertLeft(0, migratedContent);
+        ThemingApiMigration.migratedFileCount++;
       }
     }
   }
+
+  /** Logs out the number of migrated files at the end of the migration. */
+  static globalPostMigration(_tree: unknown, context: SchematicContext): void {
+    const count = ThemingApiMigration.migratedFileCount;
+
+    if (count > 0) {
+      context.logger.info(`Migrated ${count === 1 ? `${count} files` : `1 file`} to the ` +
```

Comment:
I think the ternary needs to be inversed.

Reply:
Also, out of curiosity, is there a doc/guide we could link too? that might be nice to see when the migration ran.

Reply:
I don't think that we have a doc about the migration in particular. It'll be mentioned in the changelog.